components/footer/Footer.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  Box,
+  Container,
+  Stack,
+  Text,
+  useColorModeValue,
+} from '@chakra-ui/react'
+import { FaTwitter, FaGithub } from 'react-icons/fa'
+import { SocialButton } from './SocialButton'
+
+export default function Footer() {
+  return (
+    <Box
+      // bg={useColorModeValue('#00000f', '#1d1e20')}
```

Comment:
@PBillingsby did you want to bring back the gray background, or leave it as black? 

(if leaving as black, I'd recommend remove the commented line)